src/Agent.Worker/Release/Artifacts/Definition/BuildArtifactDetails.cs

```diff
@@ -0,0 +1,30 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="BuildArtifactDetails.cs" company="Microsoft Corporation">
+//   2012-2023, All rights reserved.
+// </copyright>
+// <summary>
+//   Defines the BuildArtifactDetails type.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+using System;
+
+using Microsoft.VisualStudio.Services.Common;
+
+namespace Agent.Worker.Release.Artifacts.Definition
+{
+    public class BuildArtifactDetails : IArtifactDetails
+    {
+        public string RelativePath { get; set; }
+
+        public Uri TfsUrl { get; set; }
+
+        public VssCredentials Credentials { get; set; }
+
+        public string Project { get; set; }
+
+        public string AccessToken { get; set; }
+
+        public int ParallelDownloadLimit { get; set; }
```

Comment:
Make sense.. removed them.. we were using binfetchclient to download build artifact which requires these parameters. But thats yet to come into coreclr agent. I have removed the unused code.